src/Backend/Dockerfile

```
@@ -4,9 +4,23 @@ WORKDIR /app
 EXPOSE 8080
 EXPOSE 8081
 
-FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build   
```

Leerzeichen entfernen
```suggestion
FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
```